manim/config.py

```diff
@@ -23,67 +23,72 @@ def parse_cli():
             "-p", "--preview",
             action="store_true",
             help="Automatically open the saved file once its done",
-        ),
+        )
         parser.add_argument(
             "-w", "--write_to_movie",
             action="store_true",
             help="Render the scene as a movie file",
-        ),
+        )
         parser.add_argument(
             "-s", "--save_last_frame",
             action="store_true",
             help="Save the last frame",
-        ),
+        )
+        parser.add_argument(
+            "--dry_run", 
+            action="store_true",
+            help= "run a dry test",
```

Comment:
The space after the = can be removed, and it'd be better to described what you mean by a "dry test"

Reply:
The help string is already better (you left your review on a previous commit). I'm merging this since the extra space will be fixed once we run `black` on everything.